sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-file-handler.ts

```diff
@@ -88,20 +89,31 @@ export abstract class ECertFileHandler extends ESDCFileHandler {
     log.info(
       `Retrieving ${offeringIntensity} disbursements to generate the e-Cert file...`,
     );
-
-    const sequenceGroup = `${sequenceGroupPrefix}_${getISODateOnlyString(
+    const fileNameSequenceGroup = `${sequenceGroupPrefix}_${getISODateOnlyString(
       new Date(),
     )}`;
     let uploadResult: ECertUploadResult;
+    let fileInfo: CreateRequestFileNameResult;
```

Comment:
I would strongly advise at-least one PT E2E test covering the different sequence numbers.